tests/solvers/nn/neuraldual_test.py

```diff
@@ -55,6 +55,8 @@ class TestNeuralDual:
       back_and_forth=[True, False],
       amortization_loss=["objective", "regression"],
       conjugate_solver=[conjugate_solvers.DEFAULT_CONJUGATE_SOLVER, None],
+      tau_a=[0.8, 1.0],
```

Comment:
I think it's better to have a separate test for this, as previously the CI was failing because it took so long...

Reply:
What should the separate test look like? Just less tested options so it doesn't take as long? Should I just remove this or also add the new unbalanced neural dual test?

Reply:
Removed it for now.